.travis.yml

```diff
@@ -1,19 +1,19 @@
 language: node_js
-branches:
-  only:
-    - master
-    - /^greenkeeper/.*$/
 cache:
-  yarn: true
   directories:
+    - ~/.npm
     - node_modules
```

Comment:
As of this [SO Answer](https://stackoverflow.com/questions/42521884/should-i-have-travis-cache-node-modules-or-home-npm), only `~/.npm` should be used. I experienced errors with this configuration.